sweagent/environment/swe_env.py

```diff
@@ -373,8 +373,10 @@ def reset(self, index: int | None = None, apply_test_patch: bool = False) -> tup
         # Call install environment helper function if specified
         if self.install_environment:
             self.install_env()
-        # Install mypy for linting purposes
+        # Install linters
         self.communicate_with_handling("pip install flake8", error_msg="Failed to install flake8 (lint library)")
+        self.communicate_with_handling("npm install -g eslint", error_msg="Failed to install eslint (lint library)")
```

Comment:
I think the reason we install flake8 directly after the container is initialized is because we switch to a conda env (I think it might actually be better to install it in a global python env and then specify the abspath to there, but not important).

But with eslint etc., there's no reason to not directly install it to the docker, right?

These commands will run for every new instance, so we really want to have as few as possible here.